Content.Shared/CCVar/CCVars.cs

```diff
@@ -869,5 +869,15 @@ public sealed class CCVars : CVars
         /// </summary>
         public static readonly CVarDef<int> ResourceUploadingStoreDeletionDays =
             CVarDef.Create("netres.store_deletion_days", 30, CVar.SERVER | CVar.SERVERONLY);
+
+        /*
+         * Controls
+         */
+
+        /// <summary>
+        /// Deadzone for drag-drop interactions.
+        /// </summary>
+        public static readonly CVarDef<float> DragDropDeadZone =
+            CVarDef.Create("control.drag_dead_zone", 12f, CVar.CLIENTONLY | CVar.ARCHIVE);
```

Comment:
If we're checking UI position should scale be considered?

Reply:
Apparently this is just in screen coordinates, so AFAIK this should just be the # of pixels that the mouse actually moved on screen, and given that this is all about avoiding accidental drag interactions due to unintended mouse movements, I think it probably should be independent of UI & viewport scale anyways.